src/modules/simulator/simulator.cpp

```diff
@@ -95,6 +107,8 @@ static void usage()
 	PX4_INFO("Start simulator:     simulator start");
 	PX4_INFO("Connect using UDP: simulator start -u udp_port");
 	PX4_INFO("Connect using TCP: simulator start -c tcp_port");
+	PX4_INFO("Connect using UDP: simulator start -u udp_port hostname");
```

Comment:
Is this use case valid / tested?

Reply:
The UDP version hasn't been tested, I don't have a simulator at this time that uses a UDP connection.

Reply:
If it is not tested, would it be possible to disable it?

Reply:
Yes, I can remove it for now or comment it out. Which is preferred?

Reply:
Can you also make it so that you support passing an explicit ip address instead of hostname ?  In some cases (like windows subsystem for linux WSL2) the host name is the same for the linux and windows machines, but the ip addresses are different.  Explicit ip address can also be handy to select a specific ethernet interface which can be handy when there's multiple ways to reach the host (e.g. wifi versus hard wired).